@xen-orchestra/web/src/components/host/system/HostSystemGeneralInformation.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <UiCard>
+    <UiTitle>
+      {{ $t('general-information') }}
+    </UiTitle>
+    <VtsCardRowKeyValue>
+      <template #key> {{ $t('name') }}</template>
+      <template #value> {{ host.name_label }}</template>
+    </VtsCardRowKeyValue>
+    <VtsCardRowKeyValue>
+      <template #key>
+        {{ $t('id') }}
+      </template>
+      <template #value>
+        {{ host.id }}
+      </template>
+    </VtsCardRowKeyValue>
+    <VtsCardRowKeyValue>
+      <template #key>
+        {{ $t('description') }}
+      </template>
+      <template #value>
+        {{ host.name_description }}
```

Comment:
If there is no description, a dash should be displayed (this does not appear to be the case if description is an empty string)